### PR TITLE
WY updates

### PR DESCRIPTION
**Pull request by FishheadNate**
Added year fields to all of the jsons and source field to the jsons for the new hunting data for 2017. 

Also corrected name field to be uniform and edited properties name fields to match how the areas are labeled in the WY hunting regulations.

### CODE REVIEW

File: sources/US/WY/antelope-2017.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      "antelope"
+   ],
+   "attribution": "Wyoming Game and Fish Department",
+   "properties": {
+      "id": "HUNTAREA",
+      "name": "HUNTAREA"
+   },
+   "country": "US",
+   "state": "WY",
+   "filetype": "shp",
```
Comment (JesseCrocker) [inline sources/US/WY/antelope-2017.json]: filetype is shp, but file is geojson

File: sources/US/WY/bighornsheep-2017.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      "big horn sheep"
+   ],
+   "attribution": "Wyoming Game and Fish Department",
+   "properties": {
+      "id": "HUNTAREA",
+      "name": "HUNTAREA"
+   },
+   "country": "US",
+   "state": "WY",
+   "filetype": "shp",
```
Comment (JesseCrocker) [inline sources/US/WY/bighornsheep-2017.json]: filetype is shp, but file is geojson

File: sources/US/WY/bison-2017.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      "bison"
+   ],
+   "attribution": "Wyoming Fish and Wildlife",
+   "properties": {
+      "id": "HUNTAREA",
+      "name": "HUNTAREA"
+   },
+   "country": "US",
+   "state": "WY",
+   "filetype": "shp",
```
Comment (JesseCrocker) [inline sources/US/WY/bison-2017.json]: filetype is shp, but file is geojson

File: sources/US/WY/bobcat-2015.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+      "bobcat"
+   ],
+   "attribution": "Wyoming Game and Fish Department",
+   "properties": {
+      "id": "MGMTAREA",
+      "name": "MGMTAREA"
+   },
+   "country": "US",
+   "state": "WY",
+   "filetype": "shp",
```
Comment (JesseCrocker) [inline sources/US/WY/bobcat-2015.json]: filetype is shp, but file is geojson

File: sources/US/WY/elk-2017.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+      "elk"
+   ],
+   "attribution": "Wyoming Fish and Wildlife",
+   "properties": {
+      "id": "HUNTAREA",
+      "name": "HUNTAREA"
+   },
+   "country": "US",
+   "state": "WY",
+   "filetype": "shp",
```
Comment (JesseCrocker) [inline sources/US/WY/elk-2017.json]: filetype is shp, but file is geojson

File: sources/US/WY/mountainlion-2016.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+      "Mountain Lion"
+   ],
+   "attribution": "Wyoming Game and Fish Department",
+   "properties": {
+      "id": "HUNTAREA",
+      "name": "HUNTAREA"
+   },
+   "country": "US",
+   "state": "WY",
+   "filetype": "shp",
```
Comment (JesseCrocker) [inline sources/US/WY/mountainlion-2016.json]: filetype is shp, but file is geojson

File: sources/US/WY/muledeer-2017.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+      "mule deer"
+   ],
+   "attribution": "Wyoming Game and Fish Department",
+   "properties": {
+      "id": "HUNTAREA",
+      "name": "HUNTAREA"
+   },
+   "country": "US",
+   "state": "WY",
+   "filetype": "shp",
```
Comment (JesseCrocker) [inline sources/US/WY/muledeer-2017.json]: filetype is shp, but file is geojson

File: sources/US/WY/pheasant-2015.json [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+      "pheasant"
+   ],
+   "attribution": "Wyoming Game and Fish Department",
+   "properties": {
+      "id": "HUNTAREA",
+      "name": "HUNTAREA"
+   },
+   "country": "US",
+   "state": "WY",
+   "filetype": "shp",
```
Comment (JesseCrocker) [inline sources/US/WY/pheasant-2015.json]: filetype is shp, but file is geojson

File: sources/US/WY/whitetaileddeer-2017.json [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+      "White Tailed Deer"
+   ],
+   "attribution": "Wyoming Game and Fish Department",
+   "properties": {
+      "id": "HUNTAREA",
+      "name": "HUNTAREA"
+   },
+   "country": "US",
+   "state": "WY",
+   "filetype": "shp",
```
Comment (JesseCrocker) [inline sources/US/WY/whitetaileddeer-2017.json]: filetype is shp, but file is geojson

**Comment by FishheadNate** [comment]
changed filetype field to match filetype

**Pull request merged**